Wikipedia/Code/WMFWelcomeAnimationViewControllers.swift

```diff
@@ -0,0 +1,85 @@
+class WMFWelcomeAnimationViewController: UIViewController {
+    var welcomePageType:WMFWelcomePageType = .intro
+    fileprivate var hasAlreadyAnimated = false
+    
+    open lazy var animationView: WMFWelcomeAnimationView? = nil
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let animationView = animationView else {return}
```

Comment:
There should always be a newline for braces, especially in a guard. It needs to be easy to spot (and set a breakpoint inside of) an early return.

Reply:
Fixed.